examples/kaguya_isis_cmp.ipynb

```diff
@@ -0,0 +1,268 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Writing out a USGSCSM ISD from a PDS3 Kaguya image"
```

Comment:
Not a big deal, but we should probably change the titles on these notebooks

Reply:
Do you have any thoughts on a good way to name these?